### DISCUSSION TITLE
Scheduled report does not run after netbox containers/pods restart

### DISCUSSION THREAD

**Discussion by winadm87** [netbox-community/netbox#16155] [Help Wanted!]
### Deployment Type

Self-hosted

### NetBox Version

v3.6.4

### Python Version

3.10

### Steps to Reproduce

1. Create a netbox report
2. Schedule report to recurring for 5 min
3. Wait for first schedule iteration, report will run and make a schedule for next run in next 5 min
4. Restart netbox pods/containers, e.g. `docker compose restart`

### Expected Behavior

Netbox pods/containers restart should not affect scheduled reports

### Observed Behavior

Go back to UI and observe that scheduled job pass next iteration time without execution and do not re-schedule itself.
We can observe the same behavior as for "local docker compose deployment" and for k8s deployment.

**Comment by ziggekatten** [comment]
Same thing with 3.7.x as well as for scripts. After every version upgrade we have to reschedule all our scripts. Annoying.

**Comment by rizlas** [comment]
I can confirm that this is still happening.
Steps to reproduce are correct.

![immagine](https://github.com/user-attachments/assets/2228be06-6819-44b6-aeef-3da95b56baca)

**Reply by alehaa** [reply]
Interesting. I use the docker deployment of NetBox and didn't observe any problems all the way from 3.6 to 4.0.8. However, there may be problems if Redis is somehow reset and no longer in sync with the PostgreSQL database. Since the job function and its argument are only passed to Redis, not the `Job` model, NetBox is not able to reschedule a `Job` in this case.

**Reply by rizlas** [reply]
Really strange. I'm using the same docker compose from netbox-docker
